src/main/java/ch/njol/skript/expressions/ExprRound.java

```diff
@@ -13,44 +9,64 @@
 import ch.njol.skript.lang.Expression;
 import ch.njol.skript.lang.ExpressionType;
 import ch.njol.skript.lang.SkriptParser.ParseResult;
+import ch.njol.skript.lang.SyntaxStringBuilder;
+import ch.njol.skript.util.Patterns;
 import ch.njol.util.Kleenean;
 import ch.njol.util.Math2;
+import org.bukkit.event.Event;
+import org.jetbrains.annotations.Nullable;
 
-/**
- * @author Peter Güttinger
- */
 @Name("Rounding")
 @Description("Rounds numbers normally, up (ceiling) or down (floor) respectively.")
-@Examples({"set {var} to rounded health of player",
-		"set line 1 of the block to rounded \"%(1.5 * player's level)%\"",
-		"add rounded down argument to the player's health"})
+@Examples({
+	"set {var} to rounded health of player",
+	"set line 1 of the block to rounded \"%(1.5 * player's level)%\"",
+	"add rounded down argument to the player's health"
+})
 @Since("2.0")
 public class ExprRound extends PropertyExpression<Number, Long> {
+
+	private enum RoundType {
+		FLOOR, ROUND, CEIL
+	}
+
+	private final static Patterns<RoundType> patterns = new Patterns<>(new Object[][]{
+		{"[a|the] (round[ed] down|floored) %numbers%", RoundType.FLOOR},
+		{"%numbers% (round[ed] down|floored)", RoundType.FLOOR},
+		{"[a|the] round[ed] %numbers%", RoundType.ROUND},
+		{"%numbers% round[ed]", RoundType.ROUND},
+		{"[a|the] (round[ed] up|ceiled) %numbers%", RoundType.CEIL},
+		{"%numbers% (round[ed] up|ceiled)", RoundType.CEIL}
```

Comment:
```suggestion
		{"[a|the] (round[ed] up|ceil[ing]ed) %numbers%", RoundType.CEIL},
		{"%numbers% (round[ed] up|ceil[ing]ed)", RoundType.CEIL}
```
honestly i'm not sold on it being an option at all, but it should at least allow ceilinged